ansible/roles/host-lets-encrypt-certs-certbot/tasks/main.yml

```diff
@@ -114,8 +134,8 @@
     - "{{ _certbot_dir }}/config"
     - "{{ _certbot_dir }}/work"
     - "{{ _certbot_dir }}/logs"
-    - "{{ _certbot_dir }}/renewal-hooks"
-    - "{{ _certbot_dir }}/renewal-hooks/deploy"
+    #- "{{ _certbot_dir }}/renewal-hooks"
```

Comment:
Jim moved these up to the callers. We need to check if they moved them up to `roles/ocp4-workshop-enable-lets-encrypt-certificates`